src/node_report.cc

```diff
@@ -694,9 +731,20 @@ void PrintNativeStack(std::ostream& out) {
     return;
   }
 
+#ifdef __MVS__
+  char **res = backtrace_symbols(frames, size);
+  if (!res)
+    return;
+  int start_index = 0;
+#else
   // Print the native frames, omitting the top 3 frames as they are in node-report code
-  // backtrace_symbols_fd(frames, size, fileno(fp));
-  for (int i = 2; i < size; i++) {
+  int start_index = 2;
+#endif
+  for (int i = start_index; i < size; i++) {
```

Comment:
Since this is the only common line between `__MVS__` and non-`__MVS__` I think the code would be easier to read if the was just a single `#ifdef __MVS__`...`#else` block for this function.

Reply:
@richardlau I checked-in changes to address your 2 comments: https://github.com/nodejs/node-report/compare/master...ibmruntimes:zos?expand=1 - I realize this PR has already been approved; should we apply those updates here?

Reply:
@gabylb Since this pull request was opened against the `zos` branch of https://github.com/ibmruntimes/node-report/ any changes made there (such as the changes you've referenced) are automatically reflected in the pull request (until the pull request is either merged or closed).